lib/contents.php

```diff
@@ -233,7 +233,7 @@ function getContents($url, $header = array(), $opts = array(), $returnHeader = f
  * when returning plaintext.
  * @param string $defaultSpanText Specifies the replacement text for `<span />`
  * tags when returning plaintext.
- * @return string Contents as simplehtmldom object.
+ * @return false|simple_html_dom Contents as simplehtmldom object.
```

Comment:
Not related to Facebook bridge

Reply:
Shall I file another PR for that little type hint fix then? I'm not sure why it was labelled as returning a string tbh, but it makes my IDE sad